public-ui/src/app/views/quick-search/quick-search.component.ts

```diff
@@ -13,140 +13,144 @@ import {DomainInfosAndSurveyModulesResponse} from '../../../publicGenerated/mode
 
 
 @Component({
-  selector: 'app-quick-search',
-  templateUrl: './quick-search.component.html',
-  styleUrls: ['../../styles/template.css', '../../styles/cards.css', './quick-search.component.css']
+    selector: 'app-quick-search',
+    templateUrl: './quick-search.component.html',
+    styleUrls: ['../../styles/template.css', '../../styles/cards.css',
+        './quick-search.component.css']
 })
 export class QuickSearchComponent implements OnInit, OnDestroy {
-  pageImage = '/assets/db-images/man-standing.png';
-  title = 'Quick Guided Search';
-  subTitle = 'Enter a keyword or data standards code (eg ICD, SNOMED) in the search bar ' +
-      'to search across Electronic Health Record (EHR) data and program surveys.';
-  searchResults = [];
-  domainResults = [];
-  surveyResults = [];
-  searchText: FormControl = new FormControl();
-  prevSearchText = '';
-  totalParticipants;
-  loading = true;
-  dataType = null;
-  EHR_DATATYPE = 'ehr';
-  SURVEY_DATATYPE = 'surveys';
+    pageImage = '/assets/db-images/man-standing.png';
+    title = 'Quick Guided Search';
+    subTitle = 'Enter a keyword or data standards code (eg ICD, SNOMED) in the search bar ' +
+        'to search across Electronic Health Record (EHR) data and program surveys.';
+    searchResults = [];
+    domainResults = [];
+    surveyResults = [];
+    totalDomainResults = [];
```

Comment:
Just save the one object DomainandSurveyResults here instead of two separate variables.